# Define the goal
float32[] vert_x
float32[] vert_y
float32[] vert_z
int32[] pts
int32 prev
int32 seq
---
# Define the result
int32[] source_corr
int32[] target_corr
float32[] tf
float32 rmse
float32 fitness
---

